Example-iOS-Swift/WebViewController.swift

```diff
@@ -104,10 +114,6 @@ class WebViewController: UIViewController, WKScriptMessageHandler {
         }
         webView.load(URLRequest(url: url))
     }
-
-    func setCallback(delegate: WebViewDelegate){
```

Comment:
I removed this as it didn't seem to be referenced by anything, and nothing seemed to break when I removed it and tested afterwards with the emulator. Any reason that this was there / should I put it back @HarrisonHough?